From d0ccc8e0f9cec5a190c8c4c56fe64aa44aef7244 Mon Sep 17 00:00:00 2001
From: Sojiro Kusunoki <sojiro.kusunoki.zj@ps.hitachi-solutions.com>
Date: Thu, 13 Sep 2018 19:23:43 +0900
Subject: [PATCH 627/909] arm64: dts: r8a7795-es1-salvator-x: Add definition
 for vspm/vsp2drifer

commit b67bf25a9bf489fb93078f96567fa307f9b8f90a from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Signed-off-by: Sojiro Kusunoki <sojiro.kusunoki.zj@ps.hitachi-solutions.com>
Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/renesas/r8a7795-es1-salvator-x.dts    | 24 +++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1-salvator-x.dts b/arch/arm64/boot/dts/renesas/r8a7795-es1-salvator-x.dts
index 02f0f16f9da1..8430f0e57015 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1-salvator-x.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1-salvator-x.dts
@@ -76,6 +76,10 @@
 	mmngrbuf {
 		compatible = "renesas,mmngrbuf";
 	};
+
+	vspm_if {
+		compatible = "renesas,vspm_if";
+	};
 };
 
 &du {
@@ -208,3 +212,23 @@
 
 	status = "okay";
 };
+
+&vspbc {
+	status = "okay";
+};
+
+&vspbd {
+	status = "okay";
+};
+
+&vspi0 {
+	status = "okay";
+};
+
+&vspi1 {
+	status = "okay";
+};
+
+&vspi2 {
+	status = "okay";
+};
-- 
2.17.1

